roles/zabbix_agent/defaults/main.yml

```diff
@@ -88,7 +88,7 @@ zabbix_agent_interfaces:
     useip: "{{ zabbix_useuip }}"
     ip: "{{ zabbix_agent_ip }}"
     dns: "{{ ansible_fqdn }}"
-    port: "{{ (zabbix_agent2 == True) | ternary(zabbix_agent2_listenport, zabbix_agent_listenport) }}"
+    port: "{{ zabbix_agent_listenport }}"
```

Comment:
This change fixes the error:
```
fatal: [host.domain.io -> zabbix.domain.io]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: [{'type': 1, 'main': 1, 'useip': '{{ zabbix_useuip }}', 'ip': '{{ zabbix_agent_ip }}', 'dns': '{{ ansible_fqdn }}', 'port': '{{ (zabbix_agent2 == True) | ternary(zabbix_agent2_listenport, zabbix_agent_listenport) }}'}]: 'zabbix_agent2_listenport' is undefined. 'zabbix_agent2_listenport' is undefined. [{'type': 1, 'main': 1, 'useip': '{{ zabbix_useuip }}', 'ip': '{{ zabbix_agent_ip }}', 'dns': '{{ ansible_fqdn }}', 'port': '{{ (zabbix_agent2 == True) | ternary(zabbix_agent2_listenport, zabbix_agent_listenport) }}'}]: 'zabbix_agent2_listenport' is undefined. 'zabbix_agent2_listenport' is undefined\n\nThe error appears to be in '/path/to/collections/ansible_collections/community/zabbix/roles/zabbix_agent/tasks/api.yml': line 15, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"API | Create a new host or update an existing host's info\"\n  ^ here\n"}
```
During task:
```
TASK [community.zabbix.zabbix_agent : API | Create a new host or update an existing host's info]
```

Reply:
yep apparently I missed this one when I was updating the role.